CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Changelog
 
 ### Features
 
+- Added Japanese translation to the external currencies list ([PR 2497](https://github.com/input-output-hk/daedalus/pull/2497))
+- Added rewards address in Rewards Delegation Screen ([PR 2475](https://github.com/input-output-hk/daedalus/pull/2475))
```

Comment:
- _Added Japanese translation to the external currencies list ([PR 2497](https://github.com/input-output-hk/daedalus/pull/2497))_ - This should be moved to the **vNext**
- _Added rewards address in Rewards Delegation Screen ([PR 2475](https://github.com/input-output-hk/daedalus/pull/2475))_ - This should be removed

Reply:
Fixed in https://github.com/input-output-hk/daedalus/pull/2497/commits/c457640f73269722b63352452754b33689b55dd3